op-chain-ops/genesis/helpers.go

```diff
@@ -72,7 +73,8 @@ func AddressToCodeNamespace(addr common.Address) (common.Address, error) {
 }
 
 func IsL2DevPredeploy(addr common.Address) bool {
-	return bytes.Equal(addr[0:2], []byte{0x42, 0x00})
+	var _, celoPredeploy = predeploys.CeloPredeploys[addr.String()]
+	return celoPredeploy || bytes.Equal(addr[0:2], []byte{0x42, 0x00})
```

Comment:
Couldn't we just do
```suggestion
	var _, isPredeploy = predeploys.Predeploys[addr.String()]
	return isPredeploy
```
?

Reply:
Unfortunately this doesnt work since original Predeploys have names of contracts as keys but CeloPredeploys use addresses as Keys